cmp/form/field/BaseFieldModel.js

```diff
@@ -138,9 +140,16 @@ export class BaseFieldModel {
         this.value = v;
     }
 
+    /** @member {String[]} - all validation errors for this field. */
+    @computed
+    get errors() {
+        return compact(flatten(this._errors));
+    }
+
+
     /** @member {String[]} - all validation errors for this field and its sub-forms. */
     get allErrors() {
-        return this.errors || [];
+        return this.errors;
```

Comment:
Are these intentionally the same, i.e. are we maintaining both of these getters to avoid a breaking API change? The different docs make me think there should be a difference.

Reply:
Hi -- there was always an observable `errors` property,  just now implemented as a computed getter for technical reasons.  Differs from `allErrors` as documented, the former includes sub-form errors for this field as well as errors on
the field itself.

Reply:
But `allErrors` just returns `errors`....

Reply:
Not in the sub-class SubformsFieldModel